diff -Naurp libtar/lib/extract.c libtar.oden/lib/extract.c
--- libtar/lib/extract.c	2013-10-09 12:59:44.000000000 -0400
+++ libtar.oden/lib/extract.c	2013-10-10 01:56:17.000000000 -0400
@@ -21,6 +21,7 @@
 #include <utime.h>
 
 #ifdef STDC_HEADERS
+# include <string.h>
 # include <stdlib.h>
 #endif
 
diff -Naurp libtar/libtar/libtar.c libtar.oden/libtar/libtar.c
--- libtar/libtar/libtar.c	2013-10-09 12:59:44.000000000 -0400
+++ libtar.oden/libtar/libtar.c	2013-10-10 01:56:17.000000000 -0400
@@ -21,6 +21,7 @@
 
 #ifdef STDC_HEADERS
 # include <string.h>
+# include <stdlib.h>
 #endif
 
 #ifdef HAVE_UNISTD_H
